satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -2,164 +2,164 @@
 
 //--- Value Attribution ---//
 model value_attribution (
-	key project_id bucket_name
+    key project_id bucket_name
```

Comment:
Don't make other changes in the same PR.

Reply:
that was unintentional . I will revert it.